packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.routing.js

```diff
@@ -19,6 +19,9 @@ export default /* @ngInject */ ($stateProvider) => {
           projectId,
         }),
       breadcrumb: () => false,
+      goBacktoGeneralInformation: /* @ngInject */ ($state) => () => {
```

Comment:
```suggestion
      goBackToGeneralInformation: /* @ngInject */ ($state) => () => {
```

Reply:
Hi @mohammed-zahaf,
This is already QA done so i will take care of the breaking changes if any. I think we can ignore this for now ?
Thanks,